lib/api/components.php

```diff
@@ -41,7 +42,7 @@ function beans_load_api_components( $components ) {
 		'image'        => $root . 'image/functions.php',
 		'compiler'     => array(
 			$root . 'compiler/functions.php',
-			$root . 'compiler/class-compiler.php',
+			$root . 'compiler/class-beans-compiler.php',
```

Comment:
Just changing the filename here without renaming the original file in the Compiler API will break Beans.  Why?  That file doesn't exist. Right?

This change is included in PR #108 [on this line](https://github.com/Getbeans/Beans/pull/108/files#diff-cb462ef94df4e25483f071f7ad68cefeR44).

BTW Renaming the file will require you to make it fully compliant.  That's handle in PR #108.